.changeset/fifty-parrots-live.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
I think that a must-have before merging this is to have some kind of SSR guidance.